R/tidyverse.R

```diff
@@ -198,7 +198,7 @@ use_tidy_issue_template <- function() {
   use_directory(".github", ignore = TRUE)
   use_template(
     "tidy-issue.md",
-    ".github/ISSUE_TEMPLATE.md"
+    ".github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md"
```

Comment:
Maybe the file should be in lower case, like in the [github documentation](https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository)? Seems a bit less shouty :)

Reply:
I just kept it the same as what is was but I am happy to change it if you want.

Reply:
What do you think @batpigandme?

Reply:
For sure! Their old rec was to "yell" `ISSUE_TEMPLATE`, so I'm happy to see it lowercase-ified!